### PR TITLE
Support Ubuntu 20.04

### PR DESCRIPTION
**Pull request by MarkGibbons**
# Description

Describe what this change achieves

## Issues Resolved

List any existing issues this PR resolves

## Check List

- [ ] All tests pass. See TESTING.md for details.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a CHANGELOG entry.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: recipes/debian.rb [hunk 1]
```diff
@@ -16,8 +16,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apt_repository 'atom-ppa' do
-  uri 'ppa:webupd8team/atom'
+# apt_repository 'atom-ppa' do
+# uri 'ppa:webupd8team/atom'
+# end
```
Comment (ramereth) [inline recipes/debian.rb:21]: If we're no longer using this repo, then we should remove it from the code and update the test to check the new repo.

**Review by ramereth — CHANGES_REQUESTED** [review]

**Comment by MarkGibbons** [comment]
I fixed up the test and removed the left. over comments.  I think this one is ready.

**Comment by MarkGibbons** [comment]
Added 2104

**Comment by ramereth** [comment]
> Can we add 21.04 support too whilst you're there?

I really don't like the idea of adding non-LTS releases to testing as it's a pain to update every 6 months on every cookbook

**Comment by MarkGibbons** [comment]
Added 21.04 to kitchen.yml.  Should be ready.

**Review by damacus — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [0.5.0](https://github.com/sous-chefs/atom/releases/tag/0.5.0)

**Pull request merged**